src/Parser/Cif.php

```diff
@@ -363,7 +363,9 @@ private static function parseXmlNode(\Gettext\Translations $translations, $filen
                 static::readXmlNodeAttribute($translations, $filenameRel, $node, 'custom-label', 'PageTypeComposerFormLayoutSetControlCustomLabel');
                 break;
             default:
-                throw new \Exception('Unknown tag name '.$path.' in '.$filenameRel."\n\nNode:\n".$node->ownerDocument->saveXML($node));
+                if (strpos($filenameRel, 'packages/') !== false) {
```

Comment:
yes, but I'm only throwing the exception when we aren't in a package, thus !== false.. At least it worked, but maybe I'm missing something. Will have a look later when I'm back on the dev box.